TESTING.md

```diff
@@ -88,12 +88,14 @@ integrate such data with additional information that is only used for the enviro
     "routedNetwork": "net_datacloud_r",
     "isolatedNetwork": "net_datacloud_i",
     "directNetwork": "net_datacloud_d"
+	"orgUser": "company-admin",
+	"orgUserPassword": "secretpwd"
```

Comment:
Maybe GitHub thing, but shows not indented compared to other lines above.

Reply:
fixed